core/src/plugins/stored/CMakeLists.txt

```
@@ -30,11 +30,15 @@ if(HAVE_WIN32)
   add_definitions(-DMS_WIN${WINDOWS_BITS})
 endif()
 
+include_directories(${PROJECT_SOURCE_DIR}/src/fastlz/include)
 add_subdirectory(python)
 
 add_library(autoxflate-sd MODULE autoxflate/autoxflate-sd.cc)
 set_target_properties(autoxflate-sd PROPERTIES PREFIX "")
-target_link_libraries(autoxflate-sd bareos)
+target_link_libraries(
+  autoxflate-sd bareos bareosfastlz $<$<PLATFORM_ID:Windows>:${ZLIB_LIBRARIES}>
+  $<$<PLATFORM_ID:Windows>:${WINDOWS_LIBRARIES}>
+)
```

While the generator expressions are nice, you could just do the usual "if(WIN32)" and call `target_link_libraries` again.
Something like this:
```suggestion
target_link_libraries(
  autoxflate-sd bareos bareosfastlz
)
if(HAVE_WIN32)
  target_link_libraries(autoxflate-sd ${ZLIB_LIBRARIES} ${WINDOWS_LIBRARIES})
endif()
```
However, do we really only need ZLIB_LIBRARIES on Windows?